test/integration/targets/pip/tasks/pip.yml

```
@@ -568,6 +568,28 @@
         that:
           - "version13 is success"
 
+- name: Test virtualenv command with venv formatting
+  when: ansible_python.version.major > 2
+  block:
+    - name: Clean up the virtualenv
+      file:
+        state: absent
+        name: "{{ remote_tmp_dir }}/pipenv"
+
+    # ref: https://github.com/ansible/ansible/issues/76372
+    - name: install using different venv formatting
+      pip:
+        name: "{{ pip_test_package }}"
+        virtualenv: "{{ remote_tmp_dir }}/pipenv"
+        virtualenv_command: "{{ ansible_python_interpreter ~ ' -mvenv' }}"
+        state: present
+      register: version14
+
+    - name: ensure install using virtualenv_command with venv formatting
+      assert:
+        that:
+          - "version14 is success"
```

```suggestion
          - "version14 is changed"
```
I would probably test for a change to make sure the package was actually installed.